src/chrome/content/rules/Advertising.com.xml

```diff
@@ -1,80 +1,215 @@
 <!--
 	For problematic rules, see Advertising.com-mismatches.xml.
-
-
 	For other AOL coverage, see AOL.xml.
 
-
-	Problematic subdomains:
-
-		- ace-tag *
-		- cte.ace.beta		(mismatched, CN: servedby.advertising.com)
```

Comment:
This host does still exist, so please keep a comment about it.

Reply:
cte.ace.beta is part of the -mismatched file so it is already covered there.